mqtt/mqtt-edgehub/src/edgelet/mod.rs

```diff
@@ -2,36 +2,30 @@ mod connect;
 mod workload;
 
 pub use connect::Connector;
-pub use workload::*;
+pub use workload::{CertificateResponse, ServerCertificateRequest, WorkloadClient};
```

Comment:
Where are these used?

Reply:
they are exposed (re-exported) outside of the edgelet module